medusa/post_processor.py

```diff
@@ -1230,8 +1234,8 @@ def process(self):
                     if helpers.is_file_locked(self.file_path, False):
                         raise EpisodePostProcessingFailedException('File is locked for reading')
 
-                self.post_process_action(self.file_path, dest_path, new_base_name,
-                                         app.MOVE_ASSOCIATED_FILES, app.USE_SUBTITLES and ep_obj.series.subtitles)
+                self.post_process_action(self.file_path, dest_path, new_base_name, bool(app.MOVE_ASSOCIATED_FILES),
```

Comment:
Specific reason to use a bool() here?

Reply:
`app.MOVE_ASSOCIATED_FILES` and `ep_obj.series.subtitles` both use 1 and 0 as truthy values. I think we should be using booleans internally instead, like we usually do. A better change would be to make these changes now, but for now I wanted to make it consistent.